rumydata/_base.py

```diff
@@ -163,9 +167,15 @@ class specified in the rule_type parameter
                 if rumydata.exception.debug():
                     msg += f' [DEBUG]: {str(e)}'
                 errors.append(r.rule_exception()(msg))
+        if msg_override:
```

Comment:
Instead of trying to override each error message individually, let's add a new error with the custom message. You could also include an additional parameter that controls whether we keep the original errors, or discards them and only returns the override message.